spk/beets/src/requirements-crossenv.txt

```diff
@@ -2,4 +2,4 @@
 
 MarkupSafe==2.0.1
 PyYAML==6.0
-#Pillow==8.2.0               ==> cross/pillow
+# Pillow==8.2.0               ==> cross/pillow
```

Comment:
Once #5023 is merged I recommend you to use the python310 commented solution to get Pillow to be cross-compiled using `pip` at the version you like instead of using `cross/pillow`.  It then requires the following in your `Makefile`:
```
# [Pillow]
# DEPENDS += cross/freetype cross/libjpeg cross/zlib
# WHEELS_BUILD_ARGS += [Pillow]
# WHEELS_BUILD_ARGS += build_ext
# WHEELS_BUILD_ARGS += --disable-platform-guessing
# WHEELS_BUILD_ARGS += --enable-freetype
# WHEELS_BUILD_ARGS += --enable-jpeg
# WHEELS_BUILD_ARGS += --enable-zlib
```